### PR TITLE
Extract Census Script

### PR DESCRIPTION
**Pull request by iwootten**
## Motivation

Extracting the census individual schema involves extracting from both the different regions versions of the census, concatenating the templates together and uploading to crowdin. As a manual process, it is easy to make a mistake.

## Changes

Create a script which extracts the census_individual.pot in a single step, taking a runner data/en folder as an argument.

### CODE REVIEW

File: cli/extract_census_template.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    parser.add_argument(
+        'OUTPUT_DIRECTORY',
+        help="The destination directory for the translation template"
+    )
+
+    args = parser.parse_args()
+
+    if not os.path.isdir(os.path.join(args.RUNNER_SCHEMA_DIRECTORY)):
+        print("Not a valid runner schema directory")
```
Comment (JamesGardiner) [inline cli/extract_census_template.py]: The error message here is a bit misleading - isdir only checks if the directory exists, not whether it is a valid runner schema directory.

File: cli/extract_census_template.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        exit(1)
+
+    if not os.path.isdir(args.OUTPUT_DIRECTORY):
+        print("Not a valid output directory")
```
Comment (JamesGardiner) [inline cli/extract_census_template.py]:
```suggestion
        print("Output directory does not exist")
```

**Comment by iwootten** [comment]
I was wondering if as a final step on this, it should request if it should upload to crowdin? Given people generally have difficulty locating where they should be placed, but not wanting to assume it should happen by default.

**Comment by MebinAbraham** [comment]
We should add these to the readme?.

**Pull request merged**